### PR TITLE
docs(gce): Add reference/providers/gce section.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

**Comment by duftler** [comment]
@spinnaker/google-reviewers please review.

File: reference/providers/gce.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+### Delete Load Balancer
+
+Deletes a load balancer and all of its resources.
+
+This operation is not permitted unless there are no instances associated with the load balancer.
+
+### Create Security Group
+
+Creates a firewall rule.
```
Comment (lwander) [inline reference/providers/gce.md:125]: Is it worth elaborating what you're doing with tags here?

File: reference/providers/gce.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+Deletes an instance. In most cases, the managed instance group will provision a new instance to replace the terminated
+instance.
+
+### Terminate Instance and Shrink Server Group
+
+Atomically deletes an instance and shrinks the target size of the managed instance group.
+
+### Create Load Balancer
+
+Upserts and wires together all of the necessary resources to support Network, HTTP(S), Internal or SSL load balancing.
```
Comment (lwander) [inline reference/providers/gce.md]: I think it's worth expanding this here (maybe with @jtk54's help).

File: reference/providers/gce.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+receives traffic.
+
+### Rollback
+
+[Enables](#enable) one server group and [disables](#disable) another. The disable is only initiated once the
+newly-enabled server group's instances are all determined to be healthy.
+
+### Reboot Instance
+
+Reboots an instance (a hard reset).
```
Comment (lwander) [inline reference/providers/gce.md]: Are there GCE API docs we can point to in these cases?

Comment (duftler) [reply reference/providers/gce.md]: Done.

File: reference/providers/gce.md (hunk 2 above)
Comment (duftler) [reply reference/providers/gce.md]: Done. @jtk54 PTAL.

File: reference/providers/gce.md (hunk 1 above)
Comment (duftler) [reply reference/providers/gce.md:125]: Done.

**Review by jtk54 — APPROVED** [review]
LGTM.

**Review by lwander — APPROVED** [review]

**Comment by ttomsu** [comment]
Please run `rake test` to check your links work. I just pulled down `master` and several are broken.

**Comment by lwander** [comment]
Fixing

**Comment by lwander** [comment]
https://github.com/spinnaker/spinnaker.github.io/pull/80

**Pull request merged**